Doc/whatsnew/3.7.rst

```diff
@@ -483,138 +782,458 @@ The :func:`~binascii.b2a_uu` function now accepts an optional *backtick*
 keyword argument.  When it's true, zeros are represented by ``'`'``
 instead of spaces.  (Contributed by Xiang Zhang in :issue:`30103`.)
 
+
 calendar
 --------
 
 The class :class:`~calendar.HTMLCalendar` has new class attributes which ease
 the customisation of the CSS classes in the produced HTML calendar.
 (Contributed by Oz Tiram in :issue:`30095`.)
 
+The :func:`~calendar.weekday` function now accepts any value for its *year*
+argument (previously only values from ``1`` to ``9999`` were accepted).
+(Contributed by Mark Gollahon in :issue:`28281`.)
+
+:meth:`Calendar.itermonthdates <calendar.Calendar.itermonthdates>`
+will now consistently raise an exception when a date falls outside of the
+``0001-01-01`` through ``9999-12-31`` range.  To support applications that
+cannot tolerate such exceptions, the new
+:meth:`Calendar.itermonthdays3 <calendar.Calendar.itermonthdays3>` and
+:meth:`Calendar.itermonthdays4 <calendar.Calendar.itermonthdays4>` can be used.
+The new methods return tuples and are not restricted by the range supported by
+:class:`datetime.date`.
+(Contributed by Alexander Belopolsky in :issue:`28292`.)
+
+
+collections
+-----------
+
+The :func:`collections.namedtuple` factory has been reimplemented to rely
```

Comment:
Not in this section. It is worth mentioning in two sections: optimizations and removals.